cluster/gce/config-default.sh

```diff
@@ -41,7 +41,8 @@ MASTER_TAG="${INSTANCE_PREFIX}-master"
 MINION_TAG="${INSTANCE_PREFIX}-minion"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.244.0.0/16}"
-MINION_SCOPES=("storage-ro" "compute-rw" "https://www.googleapis.com/auth/monitoring" "https://www.googleapis.com/auth/logging.write")
+MINION_SCOPES="${MINION_SCOPES:-compute-rw,monitoring,logging-write,storage-ro}"
```

Comment:
According to the man page `gcloud alpha container clusters create --help` it is `logging-write`:

    SCOPE can be either the full URI of the scope or an alias. Available aliases are:

    Alias,URI bigquery,https://www.googleapis.com/auth/bigquery
    cloud-platform,https://www.googleapis.com/auth/cloud-platform
    compute-ro,https://www.googleapis.com/auth/compute.readonly
    compute-rw,https://www.googleapis.com/auth/compute
    computeaccounts-ro,https://www.googleapis.com/auth/computeaccounts.readonly
    computeaccounts-rw,https://www.googleapis.com/auth/computeaccounts
    datastore,https://www.googleapis.com/auth/datastore
    logging-write,https://www.googleapis.com/auth/logging.write
    monitoring,https://www.googleapis.com/auth/monitoring
    sql,https://www.googleapis.com/auth/sqlservice
    sql-admin,https://www.googleapis.com/auth/sqlservice.admin
    storage-full,https://www.googleapis.com/auth/devstorage.full_control
    storage-ro,https://www.googleapis.com/auth/devstorage.read_only
    storage-rw,https://www.googleapis.com/auth/devstorage.read_write
    taskqueue,https://www.googleapis.com/auth/taskqueue
    userinfo-email,https://www.googleapis.com/auth/userinfo.email